src/librustc_mir/monomorphize/collector.rs

```diff
@@ -1183,20 +1183,20 @@ fn collect_miri<'a, 'tcx>(
 ) {
     let alloc_kind = tcx.alloc_map.lock().get(alloc_id);
     match alloc_kind {
-        Some(AllocKind::Static(did)) => {
+        Some(GlobalAlloc::Static(did)) => {
```

Comment:
```suggestion
        Some(GlobalAlloc::Static(def_id)) => {
```
Preexisting, but I find the `did` initialism to be unfriendly to those less familiar with the code-base (it was incomprehensible to me at first).

Reply:
you may want to add this to our internal lint idea list if you feel very strongly about doing this consistently. I think we could even go as far and just forbid `did` in rustc via a `clippy.toml`